### ISSUE TITLE
parties_summary: sometimes the party column is empty

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/kingfisher-views#36]
For example, in hosted kingfisher, the result of the following query has an empty party (jsonb) column, even though the data contains parties and the other columns are populated:

```sql
select * from views.parties_summary where collection_id = 621 limit 10
```

**Comment by kindly** [comment]
@duncandewhurst 

yes this looks like it is not working for records as the code has to look inside compiledReleases as well for this case.

This is easy to fix.

**Comment by kindly** [comment]
fixed in #38

**Comment by robredpath** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [kf-views bug: not providing an (optional) identifier scheme causes identifier to be blank](https://trello.com/c/2GS98Lno/212-kf-views-bug-not-providing-an-optional-identifier-scheme-causes-identifier-to-be-blank)
